build.gradle

```diff
@@ -19,7 +19,7 @@ buildscript {
         compileSdkVersion = 30
         targetSdkVersion = 30
 
-        activityVersion = '1.2.0-rc01'
+        activityVersion = '1.3.0-alpha02'
```

Comment:
This should be `1.2.0`. Muzei doesn't rely on anything in Activity 1.3 (which is only relevant to Compose apps anyways).